### ISSUE TITLE
Quick installation issue

### ISSUE THREAD

**Issue by ian-bartholomew** [perfectsense/brightspot-cms#720]
When going through the quick installation walk though, at the end, when visiting  http://localhost:9480/cms/ two of the modules have errors "Expected mime type application/octet-stream but got text/html".  

From the logs  it looks like there is an issue with Solr.

Logs:
```
[INFO] [talledLocalContainer] SEVERE: Exception starting filter SolrRequestFilter
[INFO] [talledLocalContainer] java.lang.NoClassDefFoundError: Failed to initialize Apache Solr: Could not find necessary SLF4j logging jars. If using Jetty, the SLF4j logging jars need to go in the jetty lib/ext directory. For other containers, the corresponding directory should be used. For more information, see: http://wiki.apache.org/solr/SolrLogging
[INFO] [talledLocalContainer] 	at org.apache.solr.servlet.CheckLoggingConfiguration.check(CheckLoggingConfiguration.java:28)
[INFO] [talledLocalContainer] 	at org.apache.solr.servlet.BaseSolrFilter.<clinit>(BaseSolrFilter.java:31)
[INFO] [talledLocalContainer] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[INFO] [talledLocalContainer] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[INFO] [talledLocalContainer] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[INFO] [talledLocalContainer] 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[INFO] [talledLocalContainer] 	at java.lang.Class.newInstance(Class.java:442)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.DefaultInstanceManager.newInstance(DefaultInstanceManager.java:121)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.ApplicationFilterConfig.getFilter(ApplicationFilterConfig.java:258)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.ApplicationFilterConfig.<init>(ApplicationFilterConfig.java:105)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.StandardContext.filterStart(StandardContext.java:4574)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5193)
[INFO] [talledLocalContainer] 	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:725)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:701)
[INFO] [talledLocalContainer] 	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:717)
[INFO] [talledLocalContainer] 	at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:945)
[INFO] [talledLocalContainer] 	at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1768)
[INFO] [talledLocalContainer] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[INFO] [talledLocalContainer] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[INFO] [talledLocalContainer] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[INFO] [talledLocalContainer] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[INFO] [talledLocalContainer] 	at java.lang.Thread.run(Thread.java:745)
```

Client stack trace:
```
Can't render /cms/dashboardWidget/default/com.psddev.cms.tool.widget.RecentActivityWidget/bbf6360b-12a6-3c9d-8ccd-60bc3e5d9894!

Expected mime type application/octet-stream but got text/html. <!DOCTYPE html><html><head><title>Apache Tomcat/8.0.23 - Error report</title><style type="text/css">H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}.line {height: 1px; background-color: #525D76; border: none;}</style> </head><body><h1>HTTP Status 404 - /solr/select</h1><div class="line"></div><p><b>type</b> Status report</p><p><b>message</b> <u>/solr/select</u></p><p><b>description</b> <u>The requested resource is not available.</u></p><hr class="line"><h3>Apache Tomcat/8.0.23</h3></body></html> (org.apache.solr.client.solrj.impl.HttpSolrServer$RemoteSolrException)
org.apache.solr.client.solrj.impl.HttpSolrServer.executeMethod(HttpSolrServer.java:516)
org.apache.solr.client.solrj.impl.HttpSolrServer.request(HttpSolrServer.java:210)
org.apache.solr.client.solrj.impl.HttpSolrServer.request(HttpSolrServer.java:206)
org.apache.solr.client.solrj.request.QueryRequest.process(QueryRequest.java:91)
org.apache.solr.client.solrj.SolrServer.query(SolrServer.java:301)
com.psddev.dari.db.SolrDatabase$1.query(SolrDatabase.java:319)
com.psddev.dari.db.SolrDatabase$1.create(SolrDatabase.java:252)
com.psddev.dari.db.SolrDatabase$1.create(SolrDatabase.java:248)
com.psddev.dari.util.Lazy$1.run(Lazy.java:32)
com.psddev.dari.util.Once.ensure(Once.java:65)
com.psddev.dari.util.Lazy.get(Lazy.java:50)
com.psddev.dari.db.SolrDatabase.getSolrField(SolrDatabase.java:329)
com.psddev.dari.db.SolrDatabase.appendPredicate(SolrDatabase.java:660)
com.psddev.dari.db.SolrDatabase.access$200(SolrDatabase.java:54)
com.psddev.dari.db.SolrDatabase$StandardCompoundOperator.appendCompound(SolrDatabase.java:747)
com.psddev.dari.db.SolrDatabase.appendPredicate(SolrDatabase.java:645)
com.psddev.dari.db.SolrDatabase.buildQuery(SolrDatabase.java:403)
com.psddev.dari.db.SolrDatabase.readPartial(SolrDatabase.java:1348)
com.psddev.dari.db.AggregateDatabase.lambda$readPartial$7(AggregateDatabase.java:352)
com.psddev.dari.db.AggregateDatabase.read(AggregateDatabase.java:315)
com.psddev.dari.db.AggregateDatabase.readPartial(AggregateDatabase.java:352)
com.psddev.dari.db.ForwardingDatabase.readPartial(ForwardingDatabase.java:93)
com.psddev.dari.db.CachingDatabase.readPartial(CachingDatabase.java:306)
com.psddev.dari.db.ForwardingDatabase.readPartial(ForwardingDatabase.java:93)
com.psddev.dari.db.Query.select(Query.java:1564)
com.psddev.dari.db.AbstractDatabase$PaginatedIterator.hasNext(AbstractDatabase.java:531)
com.psddev.dari.db.Query.selectFiltered(Query.java:1614)
com.psddev.cms.tool.widget.RecentActivityWidget.writeHtml(RecentActivityWidget.java:100)
com.psddev.cms.tool.page.DashboardWidgetPage.doService(DashboardWidgetPage.java:85)
com.psddev.cms.tool.PageServlet.service(PageServlet.java:30)
javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
com.psddev.dari.util.RoutingFilter$ServletWrapper.service(RoutingFilter.java:162)
com.psddev.dari.util.RoutingFilter.doRequest(RoutingFilter.java:95)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.FrameFilter.doRequest(FrameFilter.java:66)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.FormFilter.doRequest(FormFilter.java:69)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:424)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.cms.tool.AuthenticationFilter.doRequest(AuthenticationFilter.java:94)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.cms.tool.RemoteWidgetFilter.doRequest(RemoteWidgetFilter.java:70)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.cms.db.FrameFilter.doRequest(FrameFilter.java:52)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:424)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.cms.db.PreviewDatabaseFilter.doRequest(PreviewDatabaseFilter.java:82)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.cms.db.AbFilter.doRequest(AbFilter.java:51)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.cms.rtc.RtcFilter.doRequest(RtcFilter.java:156)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.db.CachingDatabaseFilter.doRequest(CachingDatabaseFilter.java:63)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doRequest(AbstractFilter.java:498)
com.psddev.dari.db.ProfilingDatabaseFilter.doRequest(ProfilingDatabaseFilter.java:32)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.db.ResetFilter.doRequest(ResetFilter.java:69)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.db.WebResourceOverrideFilter.doDispatch(WebResourceOverrideFilter.java:113)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:424)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.cms.rtc.RtcFilter.doRequest(RtcFilter.java:156)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:424)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.ResourceFilter.doRequest(ResourceFilter.java:64)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.PingFilter.doRequest(PingFilter.java:38)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.HtmlApiFilter.doRequest(HtmlApiFilter.java:44)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:424)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.SessionIdFilter.doRequest(SessionIdFilter.java:35)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.PageContextFilter.doRequest(PageContextFilter.java:60)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.MultipartRequestFilter.doRequest(MultipartRequestFilter.java:35)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.StatsFilter.doRequest(StatsFilter.java:46)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.ProfilerFilter.doDispatch(ProfilerFilter.java:36)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.DebugFilter.doRequest(DebugFilter.java:237)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.DebugFilter.doDispatch(DebugFilter.java:158)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.ResourceFilter.doRequest(ResourceFilter.java:64)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.DebugFilter$SettingsOverrideFilter.doRequest(DebugFilter.java:737)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.LogCaptureFilter.doRequest(LogCaptureFilter.java:43)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.HeaderResponseFilter.doRequest(HeaderResponseFilter.java:34)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.SourceFilter.doRequest(SourceFilter.java:376)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.SourceFilter.doDispatch(SourceFilter.java:274)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.Utf8Filter.doRequest(Utf8Filter.java:91)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.cms.rtc.RtcFilter.doRequest(RtcFilter.java:156)
com.psddev.dari.util.AbstractFilter.doDispatch(AbstractFilter.java:420)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:582)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
com.psddev.dari.util.AbstractFilter$DependencyFilterChain.doFilter(AbstractFilter.java:575)
com.psddev.dari.util.AbstractFilter.doFilter(AbstractFilter.java:292)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)
org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:142)
org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:617)
org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:518)
org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1091)
org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:668)
org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1521)
org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1478)
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
java.lang.Thread.run(Thread.java:745)
Toggle Stack Traces
Parameters:
_=1462490445558
```

![screen shot 2016-05-05 at 4 22 33 pm](https://cloud.githubusercontent.com/assets/1447602/15060266/c6f39eb0-12dd-11e6-9e3f-ba0205dc92d1.png)

**Comment by jma7889** [comment]
I got the same thing. Any get around yet?

**Comment by gitastrophe** [comment]
Are you running Steps 1 & 2 from Quick Installation http://www.brightspot.com/docs/3.2/getting-started/installation#quick-installation or are you running the Development Installation?

**Comment by jma7889** [comment]
I am running the quick installation.

**Comment by hyoolim** [comment]
When we upgraded Solr, we didn't realize that it now required SLF4J library JAR files to be manually copied over to the web container. It's been fixed with #813.

**Comment by ramanshankrit** [comment]
hi @hyoolim 
i am facing the same issue and i have copied SLF4J jar also but now also the same error is coming. Please help.
